assets/scss/_section.scss

```diff
@@ -0,0 +1,37 @@
+/*doc
+---
+title: Section
+---
+Used to group sets of related components
+into discrete sections by adding padding
+around it's children.
+*/
+
+$sectionSpaceMultiple: 2;
+
+@include responsiveBase() {
+	.section {
+		padding: $base*$sectionSpaceMultiple $base 0;
+		&.section--padded {
```

Comment:
it would be good to also unnest this so `section--padded` has the same specificity as `section`. Keeping modifiers at the same specificity level as the base class means that if we declare them lower in the cascade than the base class, the behavior of conflicting properties is always predictable.

```css
.section.section--padded {
    padding: $something;
}
.section--borderedFancy {
    border-top: $size solid cornsilk;
    padding-top: $space; /* this will be overridden by the overqualified modifier class above */
}
```